test/distributed/_tensor/test_dtensor.py

```diff
@@ -542,6 +546,51 @@ def test_dtensor_save_load(self):
         reloaded_st = torch.load(buffer, weights_only=True)
         self.assertEqual(sharded_tensor, reloaded_st)
 
+    @with_comms
+    @unittest.skipIf(
+        IS_FBCODE,
+        "subprocess import torch fails with ModuleNotFoundError: No module named 'torch' in fbcode",
+    )
+    def test_dtensor_save_load_import(self):
+        for should_import in [True, False]:
+            device_mesh = self.build_device_mesh()
+            placements = [Shard(0)]
+            local_tensor = torch.randn(3, 3)
+            sharded_tensor = DTensor.from_local(local_tensor, device_mesh, placements)
+            with tempfile.NamedTemporaryFile() as f:
+                torch.save(sharded_tensor, f)
+                import_string = (
+                    "import torch.distributed.tensor" if should_import else ""
+                )
+                script = f"""
+import torch
+{import_string}
+x = torch.load("{f.name}", weights_only=True)
+"""
```

Comment:
Nit (no need for 3 line script, use semicolon as line separator. And no need to specify weigh_only as it is a default right now
```suggestion
                script = f"import torch;{import_sring};torch.load('{f.name}')"
```